app/views/course/experience_points_records/read_all_exp.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+json.rowCount @experience_points_count
+json.records @experience_points_records.includes(:course_user) do |record|
+  json.partial! 'experience_points_record', course: current_course, record: record
+  json.courseUserName record.course_user.name
```

Comment:
Dont you have `json.student` inside `experience_points_record` partial already? Same for `json.userExperienceUrl` below

Reply:
apparently these 2 fields are not used anymore (I have changed completely the usage of record.courseUserName into record.student.name, same with Url).